runtime/src/iree/vm/test/emitc/CMakeLists.txt

```diff
@@ -325,9 +346,10 @@ iree_c_module(
   H_FILE_OUTPUT
     "shift_ops.h"
   FLAGS
-    "--iree-vm-ir-to-c-module"
+    "--compile-mode=vm"
+    "--output-format=vm-c"
   TRANSLATE_TOOL
-    iree_tools_iree-translate
+    iree_tools_iree-compile
```

Comment:
These will all just be `iree-compile` after https://github.com/google/iree/pull/9135